library/core/src/task/wake.rs

```diff
@@ -782,12 +796,28 @@ impl LocalWaker {
         Self { waker }
     }
 
-    /// Creates a new `LocalWaker` that does nothing when `wake` is called.
+    /// Returns a reference to a `LocalWaker` that does nothing when used.
     ///
+    // Note!  Much of the documentation for this method is duplicated
+    // in the docs for `Waker::noop`.
+    // If you edit it, consider editing the other copy too.
+    //
```

Comment:
This is pervasively true, I don't feel it's worth a comment? If you want to fix it, then I would prefer a macro that expands into a few doc attrs, so it's actually fixed.

Reply:
I agree that it's pervasively true.  I have seen similar situations in many other places in the rust-lang/rust codebase, often with unintended divergence as we see here.  I'm afraid I don't have the tuits to engage in a campaign of docs deduplication.  I find it difficult to predict the style and taste preferences in the Rust Project.  I feel someone closer to the core of the project should demonstrate how this ought to be done, and probably the libs team should (at least implicitly, by lazy consensus) approve the idiom.  (The docs are not precisely identical, so the macro wouldn't be entirely straightforward.)

IMO the comment is very helpful, even with the remaining duplication, because it tells you *precisely where* you need to edit the clone-and-hack.  If that comment had been in place, perhaps 6f8a944ba4311cbcf5922132721095c226c6fbab wouldn't have contained the slip.  It's  fairly lightweight answer to the problem.

If you don't agree, or want to insist that I do substantially more work instead of this stopgap, I will remove these hunks from this MR branch.

Reply:
Hmm... I see. I tried to do the refactoring I originally suggested and found it more of a pain in the ass than I expected. As-is, this seems adequate. It makes me vaguely annoyed, which made me initially reluctant about it, but on reflection it is an annoyance with the preexisting state of things.